cli/cmd/install.go

```diff
@@ -259,3 +273,43 @@ func checkIstioInstallation() error {
 
 	return nil
 }
+
+func getAPIEndpoint(keptnNamespace string, serviceType string) (string, error) {
+	var endpoint, port string
```

Comment:
Regarding the `if`/`else if` statement: what if `serviceType` is something else? The function will always return
```
return "http://" + endpoint + ":" + port + "/api", nil
```

regardless of whether endpoint or port are set. I recommend returning string and error in this function, and in case that serviceType is unknown, return an error like "Unknown service-type %v".

Reply:
Cool, as I was only calling this service for NodePort and Loadbalancer type, but I will return the error.